lib/playground.dart

```diff
@@ -801,11 +801,27 @@ class Playground extends EditorUi implements GistContainer, GistController {
     });
   }
 
+  int _runCount = 0;
+
+  void _maybeShowInstallSdkDialog() {
+    _runCount++;
+    if (_runCount == 3) {
+      dialog.showOk(
+        'Install Flutter',
+        r'Success! '
+            r'<a class="link-bright" href="https://docs.flutter.dev/get-started/install?utm_campaign=dartpad_install">'
```

Comment:
A custom campaign needs all three parameters: `utm_source`, `utm_medium`, and `utm_campaign`. Could you update the URL to `?utm_source=dartpad.dev&utm_medium=referral&utm_campaign=dartpad_install`